ForgeUnity/Assets/BeardedManStudios/Scripts/Networking/Forge/Networking/Objects/NetworkObject.cs

```diff
@@ -548,6 +559,17 @@ protected virtual void OwnershipChanged()
 				ownershipChanged(Networker);
 		}
 
+		public void AssignMaster(NetworkingPlayer targetPlayer)
```

Comment:
Can we consider broadcasting this over to all clients, via a reliable buffered rpc (with replacePrevious = true), to offer clients the ability to know if an object is Master for them?

Reply:
Not sure if it matters for anyone else to know this as the value will be false for them anyway?

Reply:
Maybe do something like the owner change and let the person who became the master and the person who was the master know, don't think anyone else needs to know this, especially that clients don't know about the other clients just themselves and the server.